server/service/assessment/ndp/ndpExporter.ts

```diff
@@ -23,7 +23,7 @@ export const normalizeValue = R.pipe(
 )
 
 export const getCountryData = async (country: any) => {
-  const dataPoints = await NdpRepository.listOriginalDataPoints(country.countryIso)
+  const dataPoints = await OdpService.listOriginalDataPoints(country.countryIso)
```

Comment:
this exporter should be moved under service?

Reply:
separate issue